content/en/Integrations/IntegrationBuilder/How to Guides/azure-devops.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Create Azure DevOps Work Item for Findings"
+linkTitle: "Create Azure DevOps Work Item for Findings"
+weight: 40
+---
+
+## Configure Azure DevOps Integration for New Findings
+
+To set up Azure DevOps to create Work Items for new findings, follow these two main steps:
+
+1. [Create an Azure DevOps connection](#create-an-azure-devops-connection).
+2. [Create a new Azure DevOps recipe](#create-a-new-azure-devops-recipe).
+
+### Create a New Folder for Your Azure DevOps Recipes
+
+Before you begin, it's a good idea to create a new folder for all your future Azure DevOps recipes.
+
+Navigate to the Projects tab in the Integration Builder.
+
+Click the "Create" button at the top right, then select "Folder".
+
+{{% image src="/integrations/integration_builder/how_to_guides/azure-devops/01-create-folder.png" alt="Create a new folder" %}}
+
+Name the new folder and choose its location. Then click "Create folder".
+
+{{% image src="/integrations/integration_builder/how_to_guides/azure-devops/02-create-folder-modal.png" alt="Folder details" %}}
+
+You should now see the new folder in the project.
+
+### Create an Azure DevOps Connection
+
+Click the "Create" button at the top right, then select "Connection".
+
+{{% image src="/integrations/integration_builder/how_to_guides/azure-devops/03-create-connection.png" alt="Create a new connection" %}}
+
+Search for the Azure DevOps connection and click on it.
+
+{{% image src="/integrations/integration_builder/how_to_guides/azure-devops/04-select-azure-devops-connection.png" alt="Select Azure DevOps connection" %}}
+
+Fill in the connection details: name, recipe location, and select **Personal Access Token** as the OAuth 2.0 grant type. Set the API version to **7.0** and specify the Azure DevOps organization name, the email address of the user to impersonate, and the personal access token.
+
+{{% image src="/integrations/integration_builder/how_to_guides/azure-devops/05-connection-details.png" alt="Connection details" %}}
+
+> **ⓘ** Refer to the Azure DevOps [documentation](https://learn.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=Windows#create-a-pat) to create a personal access token.
+
+> **ⓘ** Use `Read, write, & manage` scopes for this example. Learn more about OAuth [scopes](https://learn.microsoft.com/en-us/azure/devops/integrate/get-started/authentication/oauth?view=azure-devops#scopes) for Azure DevOps.
+
+{{% image src="/integrations/integration_builder/how_to_guides/azure-devops/06-azure-devops-pat-scopes.png" alt="Azure DevOps PAT scopes" %}}
+
+You should now see the new connection in the project, and it is connected.
+
+{{% image src="/integrations/integration_builder/how_to_guides/azure-devops/07-connection-connected.png" alt="Connection connected" %}}
+
+### Create a New Azure DevOps Recipe
+
+Go to the "Library" tab and search for the "create work item" recipe.
```

Comment:
Searching for "work item" seems generic. Why not search for "Azure" or something Azure-specific?

Reply:
Actually "work item" is Azure-specific. 😅